### ISSUE TITLE
Table showing repo version corresponding to ipfs versions?

### ISSUE THREAD

**Issue by chriscool** [ipfs/fs-repo-migrations#18]
It would be nice if there was a table showing which repo version corresponds to which ipfs versions.
Also if fs-repo-migration gets its own version number (see issue #12), these version numbers could be added to this table too.

**Comment by whyrusleeping** [comment]
repo version '1' belongs to a version that I think you can only build using go pre1.4. repo version '2' is everything below 0.4.0, and version '3' is version 0.4.0 and on.

**Comment by jbenet** [comment]
> It would be nice if there was a table showing which repo version corresponds to which ipfs versions.

agreed.

table:
```
version 1: 0.0.0 - 0.2.3
version 2: 0.3.0 - 0.3.11
version 3: 0.4.0 - current
```
(see https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md#030---2015-04-20 for version 2)

**Comment by chriscool** [comment]
Fixed by PR #19.
